pg/lib/dialect/node_postgres_driver_dialect.ts

```diff
@@ -55,4 +56,14 @@ export class NodePostgresDriverDialect implements DriverDialect {
   getAwsPoolClient(props: pkgPg.PoolConfig): AwsPoolClient {
     return new AwsPgPoolClient(props);
   }
+
+  setupInitialProperties(props: Map<string, any>) {
+    if (props.has(WrapperProperties.KEEPALIVE_PROPERTIES.name)) {
```

Comment:
That looks a bit overengineered. There're just two properties to setup. There's almost 0 chance that these properties change any time soon.